src/status_im2/contexts/quo_preview/profile/profile_card/view.cljs

```diff
@@ -0,0 +1,110 @@
+(ns status-im2.contexts.quo-preview.profile.profile-card.view
+  (:require [quo2.components.buttons.button :as button]
+            [quo2.components.avatars.user-avatar :as user-avatar]
+            [quo2.core :as quo]
+            [quo2.foundations.colors :as colors]
+            [status-im2.contexts.quo-preview.profile.profile-card.style :as style]
+            [reagent.core :as reagent]
+            [status-im2.contexts.quo-preview.preview :as preview]
+            [react-native.core :as rn]))
+
+(defn profile-card
```

Comment:
Please use separate namespace for component and preview.
This should be moved to `quo2.components.profile.profile_card`

Reply:
I think you wanted me to tagg you here right? @J-Son89